joern-cli/frontends/pysrc2cpg/src/main/scala/io/joern/pysrc2cpg/PythonTypeHintCallLinker.scala

```diff
@@ -18,4 +22,42 @@ class PythonTypeHintCallLinker(cpg: Cpg) extends XTypeHintCallLinker(cpg) {
     case typ => typ
   }
 
+  override def linkCalls(builder: DiffGraphBuilder): Unit = {
+    val methodMap = mutable.HashMap.empty[String, MethodBase]
+    val callerAndCallees = calls
+      .map(call => (call, calleeNames(call)))
+      .toList
+    // Gather all method nodes and/or stubs
+    callerAndCallees
+      .foreach { case (call, methodNames) =>
+        val ms = callees(methodNames).l
+        if (ms.nonEmpty) {
+          ms.foreach { m => methodMap.put(m.fullName, m) }
+        } else {
+          val mNames = ms.map(_.fullName).toSet
+          methodNames
+            .filterNot(mNames.contains)
+            .map(fullName => createMethodStub(fullName, call, builder))
+            .foreach { m => methodMap.put(m.fullName, m) }
+        }
+      }
+    // Link edges to method nodes
+    callerAndCallees.foreach { case (call, methodNames) =>
+      methodNames
+        .flatMap(methodMap.get)
+        .foreach { m => builder.addEdge(call, m, EdgeTypes.CALL) }
+      if (methodNames.size == 1) {
+        builder.setNodeProperty(call, PropertyNames.METHOD_FULL_NAME, methodNames.head)
+      } else if (methodNames.size > 1) {
+        val methodNamesFiltered = methodNames.filter(!isDummyType(_))
+        val builtInMethodName   = methodNamesFiltered.filter(s => s.contains("__builtin"))
```

Comment:
Would probably be best to use the constant `PythonAstVisitor.builtinPrefix`. I should make this change in the type recovery too.